packages/styles/scss/patterns/listsection/_listsection.scss

```diff
@@ -0,0 +1,106 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection { 
+  .#{$prefix}--listsection {
+    color: $text-01;
+    background: $ui-background;
+    &__container {
+      padding-left: $layout-01;
+      padding-right: $layout-01;
+    }
+    &__col {
+      @include carbon--make-col-offset(4, 16);
+      @include carbon--make-col(7, 16);
+    }
+    
+    &__row {
+      @include carbon--make-row;
+    }
+    &__title {
+      @include type-style('expressive-heading-05');
+    }
+    &__content {
+      @include type-style('body-long-02');
+    }
+    &__divider {
+      border-bottom: 1px solid $ui-04;
+    }
+    @include carbon--breakpoint('sm') {
+      &__col {
+        @include carbon--make-col-offset(0, 4);
+        @include carbon--make-col(4, 4);
+        padding-left: $layout-01;
+        padding-right: $layout-01;
+      }
+      &__title {
+        @include type-style('expressive-heading-05');
+        margin-top: $layout-05;
+        margin-bottom: $layout-03;
+      }
+      &__content {
+        @include type-style('body-long-02');
+      }
+      &__divider {
+        border-bottom: 1px solid $ui-04;
+        margin-bottom: $layout-01;
+        margin-top: $layout-06;
+        &__col {
```

Comment:
Why does the divider have it's own column? Can it not sit within the column already created?